utils/docker/Dockerfile.el.9

```diff
@@ -0,0 +1,206 @@
+# Copyright 2022 Intel Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build an image of EL 9 based
+# environment for building the DAOS project.
+#
+
+# Pull base image
+ARG BASE_DISTRO=almalinux:9
+FROM $BASE_DISTRO
+LABEL maintainer="daos@daos.groups.io"
+
+# Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
+ARG CB0
+
+# Use local repo server if present
+ARG REPO_FILE_URL
+RUN set -e;                                                      \
+    if [ -n "$REPO_FILE_URL" ]; then                             \
+        cd /etc/yum.repos.d/ &&                                  \
+        curl -k -f -o daos_ci-el9-artifactory.repo.tmp           \
+             "$REPO_FILE_URL"daos_ci-el9-artifactory.repo &&     \
+        for file in *.repo; do                                   \
+            true > $file;                                        \
+        done;                                                    \
+        mv daos_ci-el9-artifactory.repo{.tmp,};                  \
+    fi;                                                          \
+    dnf --disablerepo \*epel\* -y install epel-release           \
+                                          dnf-plugins-core;      \
+    if [ -n "$REPO_FILE_URL" ]; then                             \
+        PT_REPO=daos_ci-rocky8-crb-artifactory;                  \
+    else                                                         \
+        PT_REPO=crb;                                             \
+    fi;                                                          \
```

Comment:
Any idea what **CRB** means?

Reply:
CodeReady Linux Builder from what i read and is the replacement for PowerTools.

Reply:
Interesting.  I have not really investigated EL9 in any capacity yet.  Seems like EL8 is still relatively new TBH.  I am surprised that we are at EL9 GA already.